jib-core/src/main/java/com/google/cloud/tools/jib/configuration/ContainerConfiguration.java

```diff
@@ -136,14 +137,26 @@ public Builder setEntrypoint(@Nullable List<String> entrypoint) {
       return this;
     }
 
+    /**
+     * Sets the user name (or UID) and optionally the user group (or GID) which the process in the
+     * container should run as.
+     *
+     * @param user the username and optionally the user group
```

Comment:
`the username/UID and optionally the groupname/GID`

Reply:
done